lib/rihanna/job_dispatcher.ex

```diff
@@ -63,6 +63,9 @@ defmodule Rihanna.JobDispatcher do
       :error ->
         job_failure(job, result, pg)
 
+      {:ok, due_at} ->
```

Comment:
I feel `{:renqueue, due_at}` may be clearer

Reply:
I'll make that change. Do you mind if I spell it `reenqueue` with two 'e's?

Reply:
Oh yes, that is better 😁

Sam has the ultimate say as to design btw :)